subsys/metrics/src/main/java/org/commonjava/indy/IndyMetricsNames.java

```diff
@@ -18,4 +18,12 @@
 
     public static final String TIMER_CONTENT_RETRIEVAL = "org.commonjava.indy.content.get.timer";
     public static final String TIMER_REST_REQUEST = "org.commonjava.indy.rest.inbound.timer";
+
+    public static final String INDY_METRICS_NAME_PREFIX = "org.commonjava.indy";
```

Comment:
Can we integrate the use of these prefixes / suffixes in the existing constants, so we're being consistent?